### DISCUSSION TITLE
Matter

### DISCUSSION THREAD

**Discussion by steini72** [arendst/Tasmota#18919] [Q&A]
![IMG_3957](https://github.com/arendst/Tasmota/assets/64614161/d1fd7db6-6a02-4ba1-b49c-55283f61be72)
Nach Test mit Iphone ????????
Matter Device nicht verfügbar(oder so)
3 mal probiert (3 x matter)
????? geht nicht mehr weg

**Comment by s-hadinger** [comment]
English please. 

If you want to remove the bridge, you need to go to tap on the three dots in the upper right corner, tap on the bridges, select Matter accessory and remove it

**Comment by steini72** [comment]
All who also had this problem
![IMG_3959](https://github.com/arendst/Tasmota/assets/64614161/4992754c-2355-4b22-9033-ee3f73412ec2)
